pymc3/plots.py

```diff
@@ -84,7 +84,7 @@ def histplot_op(ax, data):
         mind = np.min(d)
         maxd = np.max(d)
         step = max((maxd-mind)//100, 1)
-        ax.hist(d, bins=range(mind, maxd + 2, step), align='left')
+        ax.hist(d, bins=range(mind, maxd + 2, step), alpha=.3, align='left')
```

Comment:
Should the user not be allowed to set the alpha? You might not always want it (or want more transparency).